aws/rust-runtime/aws-credential-types/src/attributes.rs

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//! Types representing specific pieces of data contained within credentials or within token
+
+use zeroize::Zeroizing;
+
+/// Type representing a unique identifier representing an AWS account.
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct AccountId {
+    // Ensure it is zeroed in memory when dropped.
+    inner: Zeroizing<String>,
```

Comment:
Not sure we need this zeroed out, account ID is not considered sensitive.

Reply:
Wasn't sure whether we needed it, so tried being defensive initially. Thanks for the input, removed the use of `Zeroizing` in 811c6f3.